src/Fixer/Import/FullyQualifiedStrictTypesFixer.php

```diff
@@ -433,6 +433,10 @@ private function isReservedIdentifier(string $symbol): bool
             return false;
         }
 
+        if ('resource' === strtolower($symbol)) {
```

Comment:
why 'resource' is uniq?
worth to document in-code

Reply:
Because it is [soft reserved words](https://www.php.net/manual/en/reserved.other-reserved-words.php).